### PR TITLE
Add new public static method `wrap` for `Str` class and new `json` validator or `V` class

### PR DESCRIPTION
**Pull request by Awilum**
### Features
* New `Str::wrap()` method for wrapping the string with the given strings.
* New `V::json()` to check for valid json

```php
// Str::wrap()
echo Str::wrap($string, '# ', ' {.title}'); //=> # Pöst title {.title}
echo Str::wrap($string, '"'); //=> "Pöst title"

// V::json()
if (V::json('{"foo": "Foo"}')) {
  // do something...
}
```

## Ready?
- [x] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] Tests and checks all pass

### For review team
- [x] Add changes to release notes draft in Notion
- [x] ~Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)~

### CODE REVIEW

**Review by lukasbestle — DISMISSED** [review]
LGTM. Leaving the final review to @distantnative as I'm not 100% sure if it brings enough value to warrant a new `Str` method. But probably it won't hurt.

**Comment by bastianallgeier** [comment]
I think it's pretty useful and simple enough to not hurt if you don't need it :)

File: tests/Toolkit/VTest.php [hunk 1]
```diff
@@ -458,6 +458,18 @@ public function testIp()
...
+    public function testJson()
+    {
+        $this->assertTrue(V::json('{"foo": "bar"}'));
+        $this->assertTrue(V::json('{}'));
+        $this->assertTrue(V::json('[]'));
+        $this->assertFalse(V::json('{foo: bar}'));
+        $this->assertFalse(V::json('foo'));
+        $this->assertFalse(V::json([]));
+        $this->assertFalse(V::json(false));
+        $this->assertFalse(V::json(42));
```
Comment (afbora) [inline tests/Toolkit/VTest.php:470]: json validator would be better in separate PR. But if it will add to this PR, a small unit test can be added for empty string:

```suggestion
        $this->assertFalse(V::json(42));
        $this->assertFalse(V::json(''));
```

**Comment by bastianallgeier** [comment]
I agree with @afbora that a separate PR for each method would be better. But since both are pretty straight forward, I think we can merge them together.

**Review by bastianallgeier — APPROVED** [review]

**Review by afbora — APPROVED** [review]
